moveit_core/constraint_samplers/test/pr2_arm_kinematics_plugin.cpp

```diff
@@ -106,7 +106,7 @@ int PR2ArmIKSolver::CartToJnt(const KDL::JntArray& q_init, const KDL::Frame& p_i
   if (free_angle_ == 0)
   {
     if (verbose)
-      ROS_DEBUG_NAMED("pr2_arm_kinematics_plugin", "Solving with %f", q_init(0));
+      ROS_DEBUG_NAMED("constraint_samplers", "pr2_arm_kinematics_plugin", "Solving with %f", q_init(0));
```

Comment:
It would raise an error with `ROS_DEBUG_STREAM_NAMED`, but printf-style formatting is syntactically fine with three strings in the parameter list...

Reply:
However, this should finally end up in an printf-like command and gcc provides some extra checks for these...